external/fv3fit/fv3fit/train.py

```diff
@@ -53,9 +53,9 @@ def dump_dataclass(obj, yaml_filename):
 
 def main(args):
     with open(args.training_config, "r") as f:
-        training_config = fv3fit.TrainingConfig.from_dict(yaml.load(f))
+        training_config = fv3fit.TrainingConfig.from_dict(yaml.safe_load(f))
```

Comment:
Nice catch, this was only needed before because of the xarray serialization involved with the previous packer.